testData/dbData/README.txt

```diff
@@ -4,8 +4,10 @@ This folder contains the DB data that is used for running GPII in a development
 
 Each data file is corresponding to a document structure specified in the documentation of [GPII Data Model](https://wiki.gpii.net/w/Keys,_KeyTokens,_and_Preferences):
 
-* %universal/build/dbData/gpiiKeys.json: Contains all GPII keys.
-* %universal/build/dbData/prefsSafes.json: Contains all preferences safes corresponding to GPII keys.
+* %universal/build/dbData/snapset/gpiiKeys.json: Contains all GPII keys for snapset preferences safes.
```

Comment:
As these 4 files don't exist in this directory, it might be better to start a different section to explain they are generated by converting preferences data files located at `testData/preferences` and why 2 sets with different prefs safe types ("snapset" and "user") are generated.

"snapset" data set is for: 1. loading into production and staging CouchDB database; 2. when GPII runs in the dev config, this data set is loaded into the local pouchDB;

Writable "user" data set is loaded into the local pouchDB when running GPII integration test.

Take the chance to update line 1-3 too.

Reply:
Heh, this helps with [my question 2 above](https://github.com/GPII/universal/pull/626#discussion_r222070999).